app/views/scores/index.html.erb

```diff
@@ -11,11 +13,33 @@
     "embedParams": {
       "mode": "edit",
       "branding": false,
-      "appId": "5ecd978ef0aa2b501cf8cf25"
+      "appId": "5ecd978ef0aa2b501cf8cf25",
+      "controlsPosition": "top"
     },
   });
-  // You can now use the `embed` instance.
-  // Check out the docs at https://flat.io/developers/docs/embed/javascript.html
+
+  let headers = new Headers();
+  headers.set('Authorization', 'Bearer ' + '<%= current_user.flat_key%>')
+  headers.set('Content-Type', 'application/json')
+
+  document.getElementById('export-xml').addEventListener('click', function () {
```

Comment:
I used this to get the xml on the page, this method and the one below are coming from flat's javascript api